src/Umbraco.Core/Composing/Composers.cs

```diff
@@ -16,9 +17,8 @@ namespace Umbraco.Core.Composing
     /// </summary>
     public class Composers
```

Comment:
I don't understand this class. Why does it exist? Why is it stateful?

Reply:
This is the class where they work out what order to run composers in, compose after, compose before, disable etc.

Why is it stateful? dunno, v8 built in kindof a rush I guess.

Reply:
Bizarre. Hopefully it can all be deleted.

Reply:
Don’t see why not, all the internal composers can become umbraco builder extensions, then it’s just user composers ran in order of discovery.

But that’s another PR!

Reply:
You're doing [_insert any and all deities_] work!

Reply:
Umbraco has helped me pay the bills for about 10 years now, it’s about time I contributed.